{
  "uid" : "edd32b0eb7f3dcc8",
  "name" : "initializes human",
  "fullName" : "Contract: Initializable complex testing with inheritance.initializes human",
  "historyId" : "40e40e93d403193e2601a01478705a7",
  "time" : {
    "start" : 1635782503164,
    "stop" : 1635782503728,
    "duration" : 564
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "broken",
  "statusMessage" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0fb4c70>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
  "statusTrace" : "Error: Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0fb4c70>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))\n                    at Context.<anonymous> (test/proxy/utils/Initializable.test.js:52:44)\n                    at processImmediate (node:internal/timers:464:21)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Contract: Initializable complex testing with inheritance"
  }, {
    "name" : "testClass",
    "value" : "Contract: Initializable complex testing with inheritance"
  }, {
    "name" : "testMethod",
    "value" : "initializes human"
  }, {
    "name" : "package",
    "value" : "Contract: Initializable complex testing with inheritance"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "edd32b0eb7f3dcc8.json",
  "parameterValues" : [ ]
}